src/checkout/constants.ts

```diff
@@ -0,0 +1 @@
+export const PAYG_CREDIT_EXPERIMENT_ID = 'OWMWJ2XjTsW6Wny5AroCCg'
```

Comment:
Just gonna bring this up since it seems like it should be mentioned that we shouldn't have this in the UI as a string, that it should be an environment variables, and that it should live in the same place as all our other env variables

Reply:
so, should i use it using `process.env`?